app/src/main/java/org/httpsknighthacks/knighthacksandroid/MyFirebaseMessagingService.java

```diff
@@ -0,0 +1,42 @@
+package org.httpsknighthacks.knighthacksandroid;
+
+import android.app.Notification;
+import android.util.Log;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+public class MyFirebaseMessagingService extends FirebaseMessagingService {
+    private static final String TAG = "KnightHacksIsTheBest";
+
+    @Override
+    public void onMessageReceived(RemoteMessage remoteMessage) {
+
+        String title = remoteMessage.getData().get("title");
+        String body = remoteMessage.getData().get("body");
+
+        // ...
```

Comment:
Is this comment supposed to mean something?

Reply:
Oh sorry, I forgot to take that out.